components/input/input.html

```diff
@@ -0,0 +1,323 @@
+          />
+          <div class="logo-title">
+            <a href="../../index.html" id="logo-title-link">crimsonUI</a>
+          </div>
+        </div>
+        <div class="right-navbar">
+          <a href="https://github.com/omgaikwad/crimsonUI" target="_blank">
+            <i class="fab fa-github fa-2x navbar-icons"></i
+          ></a>
+
+          <a href="https://twitter.com/OmGaikwad_" target="_blank">
+            <i class="fab fa-twitter fa-2x navbar-icons"></i
+          ></a>
+
+          <a href="https://www.linkedin.com/in/omgaikwad1/" target="_blank">
+            <i class="fab fa-linkedin-in fa-2x navbar-icons"></i
+          ></a>
+        </div>
+      </nav>
+
+      <!-- SIDEBAR SECTION -->
+      <div class="sidebar" id="main-sidebar">
+        <div class="components">
+          <h3 class="components-heading">Get Started</h3>
+
+          <ul class="components-list">
+            <li class="components-list-item">
+              <a href="../../pages/introduction.html">Introduction</a>
+            </li>
+          </ul>
+        </div>
+
+        <div class="components">
+          <h3 class="components-heading">Components</h3>
+
+          <ul class="components-list">
+            <li class="components-list-item">
+              <a href="../avatar/avatar.html">Avatar</a>
+            </li>
+            <li class="components-list-item">
+              <a href="../alert/alert.html">Alert</a>
+            </li>
+            <li class="components-list-item">
+              <a href="../badge/badge.html">Badge</a>
+            </li>
+            <li class="components-list-item">
+              <a href="../button/button.html">Button</a>
+            </li>
+            <li class="components-list-item">
+              <a href="../card/card.html">Cards</a>
+            </li>
+            <li class="components-list-item">
+              <a href="../image/image.html">Image</a>
+            </li>
+            <li class="components-list-item">
+              <a href="../input/input.html" id="active-page">Input</a>
+            </li>
+          </ul>
+        </div>
+      </div>
+
+      <!-- MAIN SECTION -->
+
+      <section class="main" id="main-section">
+        <h1 class="main-heading">INPUT</h1>
+        <p>
+          Inputs are necessary for user interaction with the website. Inputs are
+          used to take information from the user, and then save the data to the
+          server, accordingly.
+        </p>
+
+        <!-- Label Inputs -->
+        <h2>Label Inputs</h2>
+        <p>
+          Label Input has a label present at the top of the input field. They
+          can be customized to show error error-message, and change color
+          accordingly.
+        </p>
+
+        <!-- Code Output -->
+        <div class="code-output-container">
+          <form>
+            <h3>Sign Up</h3>
+            <label for="">Full Name</label>
+            <input type="text" placeholder="Full Name" />
```

Comment:
dont leave the `for` attribute empty, 
good practice is to use the same name as `id` of `input` element

Reply:
I will add it before merging it